tutorial_sarscov2.md

```diff
@@ -125,106 +149,165 @@ aligner = bwa
 
 Run the V-pipe analysis (the necessary dependencies will be downloaded and installed in conda environments managed by snakemake):
 
 ```bash
-snakemake -s vpipe.snake --use-conda -p --cores 2
+./vpipe --use-conda -p --cores 2
 ```
 
+## Output
 
-## Larger dataset
+The Wiki contains an overview of the [output files](https://github.com/cbg-ethz/V-pipe/wiki/output).
+The output of the SNVs calling is aggregated in a standard [VCF](https://en.wikipedia.org/wiki/Variant_Call_Format) file, located in
+`samples/`*\{hierarchy\}*`/variants/SNVs/snvs.vcf`, you can open it with your favorite VCF tools for aggregated or downstream processing.
 
-Log onto [Euler](https://scicomp.ethz.ch/wiki/Euler) with ETH account.
+*[VCF]: Variant Call Format
 
-```bash
-ssh login.euler.ethz.ch
-```
+There is also a visual report generated in `samples/`*\{hierarchy\}*`/visualization/index.html` that you can simply open with your favorite web browser (Firefox, Chrome). You will get an output similar to that:
 
-There is a much larger test data that you can use:
+![SARS-CoV-2 report visualisation]({{ "/img/visualisation-sars-cov2.png" | relative_url }})
 
-```bash
-ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/
-ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/PRJNA6*
-ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/PRJNA6*/sample*/
+You can use the mouse selection to zoom in region and double clicks to zoom out.
+
+### Expected output
+
+The small dataset that we used in this first part has been analyzed by [doi:10.1093/nsr/nwaa036](https://doi.org/10.1093/nsr/nwaa036).
+We find their results (analyzed with bwa, samtools mpileup and bcftools) in the table 2 of the article:
+
+|Accession number|Genomic position|Ref allele|Alt allele|Ref reads|Alt reads|Location_date  |GISAID ID     |
+|:---------------+---------------:+:--------:+:--------:+--------:+--------:+:--------------+:-------------|
+|SRR10903401     |            1821|     G    |     A    |       52|        5|WH_2020/01/02.a|EPI_ISL_406716|
+|SRR10903401     |           19164|     C    |     T    |       40|       12|WH_2020/01/02.a|EPI_ISL_406716|
+|SRR10903401     |           24323|     A    |     C    |      102|       67|WH_2020/01/02.a|EPI_ISL_406716|
+|SRR10903401     |           26314|     G    |     A    |       15|        2|WH_2020/01/02.a|EPI_ISL_406716|
+|SRR10903401     |           26590|     T    |     C    |       10|        2|WH_2020/01/02.a|EPI_ISL_406716|
+|SRR10903402     |           11563|     C    |     T    |      164|       26|WH_2020/01/02.b|EPI_ISL_406717|
+
+Using the reports and zoom function, compare with the results given out by V-pipe (with bwa and ShoRAH).
+
+- For positions 19164 and 24323 of SRR10903401 and position 11563 of SRR10903402,
+  we expect to see similar results in V-pipe.
+- For the remaining positions (1821, 26314 and 26590 of SRR10903401),
+  we expect that ShoRAH will consider the variants of poor quality and reject them
```

Comment:
for the remaining positions, we expect ShoRAH to not call the SNVs because of low read coverage (<= 5 supporting the alternative).